routes/modelExport.js

```diff
@@ -1,9 +1,33 @@
-var linz = require('../');
+'use strict';
+
+const linz = require('../');
+const setTemplateScripts = require('../lib/scripts');
```

Comment:
Yeah, something like:

```
linz.api.views.getScripts = function (req, res, scripts = []) {
    return setTemplateScripts(req, res, scripts);
}

linz.api.views.getStyles = function (req, res, styles = []) {
    return setTemplateStyles(req, res, styles);
}
```

Then you'd use it like so in `routes/modelExport.js` for example:
```
Promise.all([linz.api.views.getScripts(req, res, [{ src: '/something.js' }]), linz.api.views.getScripts(req, res, [{ src: '/something.js' }])])
   .then([scripts, styles] => {
      return res.render();
   });
```

Make sense?